src/website/views/templates/autopay.pug

```diff
@@ -0,0 +1,592 @@
+---
+title: Autopay
+---
+
+- var days = ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23', '24', '25', '26', '27', '28', '29', '30', '31'];
+- var years = ['2019', '2020', '2021', '2022', '2023', '2024', '2025', '2026', '2027', '2028', '2029', '2030'];
+
+p.-text Basic Autopay page template. Indicates several ways to automate the way to pay.
+
+h4 Base
+.example.-mb--3.-bg--grey-10
+  .chi-app.-p3#autopayBase
+    .a-card.-p3
+      .a-grid.-noGutter
+        div
+          h5.-m--0.a-card__title Automatic Payments
+          p.-text.-mb--3 Each month, an automatic payment for the full amount of your bill will be paid to CenturyLink from the payment method of your choosing.
+        div.a-col.-w-md--4
+          .m-formGroup.-labeled.-mb--1
+            label.a-label(for='Base') Choose a payment method
+            .m-dropdown
+              button.a-btn.-xl.m-dropdown__trigger(id='Base', onclick='selectPayment(id)') Checking Account
+              .m-dropdown__menu
+                a.m-dropdown__menu-item(href='#checkingAccountBase', name='Checking') Checking account
```

Comment:
The dropdown isn't hiding after an option is clicked

Reply:
done

Reply:
Can we also apply -active to the menu-item when it's selected?